prototype/src/main/webapp/meeting-history.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
nit: this is indented weirdly - you'd want the opening and closing head tags to be on the same level.

Reply:
Fixed :)